### PR TITLE
[ticket/14189] Add core.gen_sort_selects_after event

### PR DESCRIPTION
**Pull request by Senky**
[PHPBB3-14189](https://tracker.phpbb.com/browse/PHPBB3-14189)

### CODE REVIEW

File: phpBB/includes/functions_content.php [hunk 1]
```diff
@@ -106,6 +106,42 @@ function gen_sort_selects(&$limit_days, &$sort_by_text, &$sort_days, &$sort_key,
...
+		'sort_key',
+		'sort_dir',
+		's_limit_days',
+		's_sort_key',
+		's_sort_dir',
+		'u_sort_param',
+		'def_st',
+		'def_sk',
+		'def_sd',
+		'sorts'
```
Comment (Nicofuma) [inline phpBB/includes/functions_content.php]: missing comma at the end of the array

**Comment by marc1706** [comment]
Will break PHP7 builds in master branch.

**Comment by Senky** [comment]
How is that possible? There is nothing different from previous PRs...
Can you aid me with CI report so that I do not need to build it again?

**Comment by Senky** [comment]
I think it's OK now, Bamboo agrees :)

**Comment by marc1706** [comment]
Merged via https://github.com/phpbb/phpbb/commit/ec70ec96e725696b2efc922ad60e802e58e97c82

**Pull request closed without merging**